pipeline_dp/combiners.py

```diff
@@ -25,3 +30,49 @@ def merge_accumulators(self, accumulator1, accumulator2):
     def compute_metrics(self, accumulator: 'Accumulator'):
         """Computes and returns the result of aggregation."""
         pass
+
+
+class CombinerParams:
+    """Parameters for an combiner.
+
+    Wraps epsilon and delta from the MechanismSpec which are lazily loaded.
+    AggregateParams are copied into a MeanVarParams instance.
+    """
+
+    def __init__(self, spec: budget_accounting.MechanismSpec,
+                 aggregate_params: pipeline_dp.AggregateParams):
+        self._mechanism_spec = spec
+        self.aggregate_params = copy.copy(aggregate_params)
+
+    @property
+    def eps(self):
+        return self._mechanism_spec.eps
+
+    @property
+    def delta(self):
+        return self._mechanism_spec.delta
+
+    @property
+    def mean_var_params(self):
+        return dp_computations.MeanVarParams(
+            self.eps, self.delta, self.aggregate_params.low,
+            self.aggregate_params.high,
+            self.aggregate_params.max_partitions_contributed,
+            self.aggregate_params.max_contributions_per_partition,
+            self.aggregate_params.noise_kind)
+
+
+class CountCombiner(Combiner):
+
+    def __init__(self, params: CombinerParams):
+        self._params = params
+
+    def create_accumulator(self, values) -> int:
+        return len(values)
```

Comment:
Why does this return length of the array? Maybe we can add some docs to the parent method to document the expectation for the return type.

Reply:
I've added more comments on Combiner based class on how the Combiner framework works and on CountCombiner class. Please check whether it becomes more clear

Reply:
Yes that's much clearer, thank you!